### DISCUSSION TITLE
How can I detect Dark Reader from javascript?

### DISCUSSION THREAD

**Discussion by KeisukeNagakawa** [darkreader/darkreader#8261] [Q&A]
Hi, I am writing a website and would like to detect whether dark-reader is used or not. Is there any way to do so?

I have searched and found some solution like using: 
```
      window.matchMedia('(prefers-color-scheme: dark)').matches
```

but it did not work for Dark Reader. Thanks!

**Comment by Gusted** [comment]
```js
const isDarkReaderDetected = [...document.querySelectorAll('style')].some((el) => el.classList.contains('darkreader'));
```
